sonarr/series.go

```diff
@@ -6,71 +6,173 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/url"
+	"path"
 	"strconv"
+	"time"
 
 	"golift.io/starr"
 )
 
+// Series the /api/v3/series endpoint.
+type Series struct {
```

Comment:
I'd like to at least keep the struct names the same. This one should be named `AddSeriesInput`, and `SeriesOutput` should be `Series`. This will prevent library consumers from having to change piles of code.

Reply:
Changing the struct names unnecessarily changes the interfaces to all of the Get* and Lookup* methods. These are arguably the most used pieces of this library. We should avoid changing interfaces where we don't need to.

Reply:
changed the two struct names as suggested